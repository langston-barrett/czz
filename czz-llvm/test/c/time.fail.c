#include <stdio.h>
#include <time.h>
int main(int argc, char *argv[]) {
  time_t t = time(NULL);
  printf("time: %lu\n", t);
  return 0;
}
