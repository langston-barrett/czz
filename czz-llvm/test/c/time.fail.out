time: 1661711109
